Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -152,4 +153,35 @@ + (NSNumber *)getScreenHeight {
     return [NSNumber numberWithInteger:(NSInteger)height];
 }
 
++ (NSDictionary *)getListOfApps {
+    NSMutableArray *appsPresent = [[NSMutableArray alloc] init];
+    NSMutableArray *appsNotPresent = [[NSMutableArray alloc] init];
+    NSDictionary *appsData = [NSDictionary dictionaryWithObjects:@[appsPresent, appsNotPresent] forKeys:@[@"canOpen", @"notOpen"]];
```

Comment:
Are you aware of the shorthand notation for dictionaries? It saves a ton of space, and is actually more readable, imo.

```
NSDict *dict = @{
    @"foo": @"bar"
};
```